src/test/java/com/training/services/tests/GuestAccountAuthenticateTest.java

```diff
@@ -0,0 +1,43 @@
+package com.training.services.tests;
+
+import com.training.base.BaseTest;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.json.simple.JSONObject;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import services.ga.AuthenticationResponse;
+import static io.restassured.RestAssured.given;
+
+public class GuestAccountAuthenticateTest extends BaseTest{
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Test
+    public void  testGuestAuthenticate(){
+        RestAssured.baseURI="https://aws-stg1.api.rccl.com/en/royal/web/v3/guestAccounts";
+        RequestSpecification request= given();
+        request.header(map.get("AppKeyHeader"),(map.get("AppKeyValue")));
+        request.header(map.get("ContentTypeHeader"),(map.get("ContentTypeValue")));
+        JSONObject requestParams = new JSONObject();
+        requestParams.put("uid", "testShrikant@api.com");
+        requestParams.put("password", "Password1");
+        request.body(requestParams.toString());
+        Response response = request.post("/authentication/login");
```

Comment:
Can you chain the whole response - in one go, rather than declaring everything and then using it here?